crates/telemetry-subscribers/src/lib.rs

```diff
@@ -184,6 +188,7 @@ pub fn init(config: TelemetryConfig) -> TelemetryGuards {
         // Output to file or to stdout with ANSI colors
         let fmt_layer = fmt::layer()
             .with_ansi(config.log_file.is_none())
+            .with_span_events(FmtSpan::NEW | FmtSpan::CLOSE)
```

Comment:
We should make this an option to disable, because it's quite verbose and adds lots of information just by itself.